Issue archiving beta (#6980)
getsentry/sentry-docs: src/docs/product/issues/states-triage/escalating-issues/index.mdx

---
title: Escalating Issues Algorithm
sidebar_order: 40
description: "Learn how the escalating issues algorithm works."

> ---

Reviewer: ```suggestion
redirect_from:
  - /product/accounts/early-adopter-features/issue-archiving
---
```

<Include name="merge-escalating-note.mdx" />

Sentry defines an issue as `Escalating` when the number of events is significantly higher than the previous week.

> The escalating issues algorithm uses historical data from the previous week to calculate thresholds for the next week on a per-issue basis. If an issue exceeds its predetermined threshold, it gets tagged as "escalating." An escalating issue is automatically surfaced in the "Unresolved" and "Escalating" tabs, even if the issue was previously archived.

Reviewer 2: Should we clarify that this only applies to issues that are marked as `archived until escalating`? The last bit leaves it open-ended.

Reviewer: Updated the wording. I thought any archived issue can be escalated unless it was "archived forever". LMK if this isn't correct.

> The escalating issues algorithm uses historical data from the previous week to calculate thresholds for the next week on a per-issue basis. If an issue exceeds its predetermined threshold, it gets tagged as "escalating." An escalating issue is automatically surfaced in the "Unresolved" and "Escalating" tabs, even if the issue was previously archived.

Reviewer: ```suggestion
The escalating issues algorithm uses historical data from the previous week to calculate thresholds for the next week on a per-issue basis. If an issue exceeds its predetermined threshold, it gets tagged as "escalating", unless it was "[Archived Forever](/product/issues/states-triage/#archive)". An escalating issue is automatically surfaced in the "Unresolved" and "Escalating" tabs.
```

---
title: Escalating Issues Algorithm
sidebar_order: 40
description: "Learn how the escalating issues algorithm works."

> redirect_from:
>   - /product/accounts/early-adopter-features/issue-archiving

Reviewer: ```suggestion
```

---

<Include name="merge-escalating-note.mdx" />

Sentry defines an issue as `Escalating` when the number of events is significantly higher than the previous week.

> The escalating issues algorithm uses historical data from the previous week to calculate thresholds for the next week on a per-issue basis. If an issue exceeds its predetermined threshold, it gets tagged as "escalating", unless it was "[Archived Forever](/product/issues/states-triage/#archive)". An escalating issue is automatically surfaced in the "Unresolved" and "Escalating" tabs.

Reviewer: ```suggestion
The escalating issues algorithm uses historical data from the previous week to calculate thresholds for the next week on a per-issue basis. If an issue exceeds its predetermined threshold, it gets tagged as `Escalating`, unless it was "[Archived Forever](/product/issues/states-triage/#archive)". An escalating issue is automatically surfaced in the "Unresolved" and "Escalating" tabs.
```
